packages/flutter_android_lifecycle/android/build.gradle

```diff
@@ -0,0 +1,42 @@
+group 'io.flutter.plugins.flutter_android_lifecycle'
+version '1.0'
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.0'
+    }
+}
+
+rootProject.allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 16
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+
+    dependencies {
+        implementation "androidx.annotation:annotation:1.1.0"
+        implementation "androidx.lifecycle:lifecycle-runtime:2.0.0-rc01"
+        api 'android.arch.lifecycle:runtime:1.1.1'
```

Comment:
These should be changed to the `androidx.` versions.

Reply:
done